src/main/java/org/team199/robot2020/subsystems/Intake.java

```diff
@@ -53,7 +53,7 @@ public void deploy() {
 
     public void retract() {
         intakePistons1.set(DoubleSolenoid.Value.kForward);
-        intakePistons2.set(DoubleSolenoid.Value.kReverse);
+        intakePistons2.set(DoubleSolenoid.Value.kForward);
```

Comment:
Are you sure this change is correct? Despite the names, intakePistons1 and intakePistons2 don't refer to 2 different pistons. They refer to 2 different solenoids. If I remember correctly, each solenoid controls flow in/out of one end of both pistons. This allows us to "do the flop" where we connect both ends of both pistons to exhaust so that they are neither pulling nor pushing.

Reply:
I believe we ran into the problem that the intake didn't deploy. This change was the fix copied from [`demo-mode`](https://github.com/DeepBlueRobotics/RobotCode2020/tree/demo-mode).